b2sdk/sync/path.py

```diff
@@ -0,0 +1,74 @@
+######################################################################
+#
+# File: b2sdk/sync/path.py
+#
+# Copyright 2021 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from abc import ABC, abstractmethod
+from typing import List
+
+from ..file_version import FileVersionInfo
+
+
+class AbstractSyncPath(ABC):
+    """
+    Represent a path in a source or destination folder - be it B2 or local
+    """
+
+    def __init__(self, relative_path: str, mod_time: int, size: int):
+        self.relative_path = relative_path
+        self.mod_time = mod_time
+        self.size = size
+
+    @abstractmethod
+    def is_visible(self):
```

Comment:
property?

Reply:
You think so? I don't really see the point of pretending this is an attribute in front of the caller

Reply:
Your call